mlflow_databricks_artifacts/store/artifact_repo.py

```diff
@@ -0,0 +1,323 @@
+import base64
+import logging
+import os
+import posixpath
+import requests
+import uuid
+
+from mlflow import get_tracking_uri
+from mlflow.entities import FileInfo
+from mlflow.exceptions import MlflowException
+from mlflow.store import artifact_repo
+from mlflow.protos.service_pb2 import MlflowService, GetRun, ListArtifacts
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, INTERNAL_ERROR
+
+try:
+    from mlflow.protos.databricks_artifacts_pb2 import (
+        DatabricksMlflowArtifactsService,
+        GetCredentialsForWrite,
+        GetCredentialsForRead,
+        ArtifactCredentialType,
+    )
+except Exception:
```

Comment:
Perhaps `ImportError` would be a more specific error to use here. I started with broadly catching "Exception" to avoid unforseen failures, but `ImportError` is probably sufficient.

Reply:
+1